dipy/tracking/mesh.py

```diff
@@ -0,0 +1,181 @@
+
+import numpy as np
+
+from dipy.utils.optpkg import optional_package
+
+_, has_fury, _ = optional_package('fury')
+
+def random_coordinates_from_surface(nb_triangles, nb_seed, triangles_mask=None,
```

Comment:
After the import above, add the following decorator above the function: `@doctest_skip_parser`